build.gradle

```diff
@@ -1,6 +1,7 @@
 buildscript {
 	repositories {
 		maven { url "https://repo.spring.io/plugins-release" }
+		maven { url "https://plugins.gradle.org/m2/"}
```

Comment:
I had to update some of the repositories as the permissions for spring repos changed in 2020.

https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020